### ISSUE TITLE
Grid with 1 item fails to render

### ISSUE THREAD

**Issue by kdar** [ogham/rust-term-grid#1]
If you only add one string to the grid, it will fail and return None.
Here is the example code:

```
extern crate term_grid;
use term_grid::{Grid, GridOptions, Direction, Filling};

fn main() {
    let mut grid = Grid::new(GridOptions {
        direction:  Direction::TopToBottom,
        filling:    Filling::Spaces(2),
    });

    for i in 0..1 {
        grid.add(format!("{}", 2_isize.pow(i)).into())
    }

    if let Some(grid_display) = grid.fit_into_width(40) {
        println!("{}", grid_display);
    }
    else {
        println!("Couldn't fit grid into 40 columns!");
    }
}
```

**Comment by ogham** [comment]
Well this *definitely* looks like a bug. Thanks for the report! I'll work on a fix.

**Comment by ogham** [comment]
Fixed in 49344de0ddfbc51e93827d2848d979821ca904f8
